CONTRIBUTING.md

```diff
@@ -53,6 +62,8 @@ dependencies are installed and the packages get linked correctly. Here a short g
 
 We suggest using [Node.js](https://nodejs.org/en/) version 16 for development purposes.
 
+After that we recommend enabling [Node.js corepack](https://nodejs.org/docs/latest-v16.x/api/corepack.html) with `corepack enable`, and [pnpm](https://pnpm.io/) with `corepack prepare pnpm --activate`.
```

Comment:
When I tested, `corepack` command was unavailable, so I had to `brew install corepack`. Do we mention/recommend this?

Reply:
When I tested, `corepack prepare pnpm --activate` threw `Usage Error: Invalid package manager specification in CLI arguments; expected a semver version, range, or tag`, so I had to remove `pnpm` from the command. Is the command correct?

Reply:
what version of node.js did you test this on?

Reply:
v16.13.1

Reply:
[Homebrew explicitly remove `npm` and `corepack` in the `node` formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/node.rb#L67). I've added a note about it this file.